core/src/stored/backends/gfapi_device.h

```diff
@@ -40,6 +40,8 @@ class gfapi_device : public Device {
  private:
   char* gfapi_configstring_;
   char* gfapi_uri_;
+  char* gfapi_logfile_;
```

Comment:
```suggestion
  char* gfapi_logfile_{};
```

Reply:
Should double check if the constructor is really called. Ohterwise this has no effect.